app/controllers/StatsControllerAJAX.php

```diff
@@ -0,0 +1,241 @@
+<?php
+  require_once($GLOBALS['dirpre'].'controllers/Controller.php');
+
+  interface StatsControllerAJAXInterface {
+    public static function getRecruiterStats();
+    public static function getStudentStats();
+    public static function getSubletStats();
+    public static function getSchoolStats();
+    public static function getMessageStats();
+    public static function getApplicationStats();
+  }
+
+  class StatsControllerAJAX
+    extends StatsController
+    implements StatsControllerAJAXInterface
+  {
+
+    public static function getRecruiterStats() {
+
+      global $MRecruiter, $MJob, $MCompany, $MApp;
```

Comment:
We should use the static class rather than the instances. I'll go ahead and change this.